From 08f63f3c0b83e3767206bf565544d091c84d5f98 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:05:55 -0400
Subject: [tools] perf/report: Use ui__has_annotation()

Message-id: <1410181752-25631-111-git-send-email-jolsa@redhat.com>
Patchwork-id: 91940
O-Subject: [PATCH RHEL7.1 BZ1134356 110/307] perf report: Use ui__has_annotation()
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit b9ce0c99d820b7680fdb4dc39bc7b5ff79d6b5b0
Author: Namhyung Kim <namhyung@kernel.org>
Date: Tue Mar 18 15:32:26 2014 +0900

description
===========
Since we introduced the ui__has_annotation() for that, don't open code
it.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-report.c b/tools/perf/builtin-report.c
index a74059f..c8f2113 100644
--- a/tools/perf/builtin-report.c
+++ b/tools/perf/builtin-report.c
@@ -928,7 +928,7 @@ repeat:
   * so don't allocate extra space that won't be used in the stdio
   * implementation.
   */
- if (use_browser == 1 && sort__has_sym) {
+ if (ui__has_annotation()) {
   symbol_conf.priv_size = sizeof(struct annotation);
   machines__set_symbol_filter(&session->machines,
          symbol__annotate_init);
-- 
1.7.1